.github/CODEOWNERS

```diff
@@ -11,7 +11,7 @@
 
 # Services
 /core/services/directrequest @smartcontractkit/keepers
-/core/services/feeds @smartcontractkit/FMS
+/core/services/feeds @smartcontractkit/op-core @eutopian @yevshev
```

Comment:
When possible we want to use team based owners over individual contributors

Reply:
Individuals are not disallowed entirely, but they are exceptions

Reply:
These are definitely exceptions for us as well - they are no longer part of OPCore but their expertise in the product (CLO) makes them extremely valuable contributors which we continue to rely on.